_overviews/scala3-book/packaging-imports.md

```diff
@@ -93,90 +125,165 @@ A note before moving on:
 
+{% tabs packaging-imports-5 %}
+{% tab 'Scala 2 and 3' %}
 ```scala
 import scala.concurrent.{Future, Promise, blocking}
 ```
+{% endtab %}
+{% endtabs %}
 
 When you want to import everything from the *scala.concurrent* package, use this syntax:
 
+{% tabs packaging-imports-6 class=tabs-scala-version %}
+{% tab 'Scala 2' %}
+```scala
+import scala.concurrent._
+```
+
+{% endtab %}
+{% tab 'Scala 3' for=packaging-imports-6 %}
+
 ```scala
 import scala.concurrent.*
 ```
 
+{% endtab %}
+{% endtabs %}
 
 ### Renaming members on import
 
 Sometimes it can help to rename entities when you import them to avoid name collisions.
 For instance, if you want to use the Scala `List` class and also the *java.util.List* class at the same time, you can rename the *java.util.List* class when you import it:
 
+{% tabs packaging-imports-7 class=tabs-scala-version %}
+{% tab 'Scala 2' %}
+```scala
+import java.util.{List => JavaList}
+```
+
+{% endtab %}
+{% tab 'Scala 3' for=packaging-imports-7 %}
+
 ```scala
 import java.util.{List as JavaList}
 ```
+{% endtab %}
+{% endtabs %}
 
 Now you use the name `JavaList` to refer to that class, and use `List` to refer to the Scala list class.
 
 You can also rename multiple members at one time using this syntax:
 
+{% tabs packaging-imports-8 class=tabs-scala-version %}
+{% tab 'Scala 2' %}
+```scala
+import java.util.{Date => JDate, HashMap => JHashMap, _}
+```
+
+{% endtab %}
+{% tab 'Scala 3' for=packaging-imports-8 %}
+
 ```scala
 import java.util.{Date as JDate, HashMap as JHashMap, *}
 ```
 
-That line of code says, “Rename the `Date` and `HashMap` classes as shown, and import everything else in the _java.util_ package without renaming any other members.”
+{% endtab %}
+{% endtabs %}
 
+That line of code says, “Rename the `Date` and `HashMap` classes as shown, and import everything else in the _java.util_ package without renaming any other members.”
 
 ### Hiding members on import
 
 You can also *hide* members during the import process.
 This `import` statement hides the *java.util.Random* class, while importing everything else in the *java.util* package:
 
+{% tabs packaging-imports-9 class=tabs-scala-version %}
+{% tab 'Scala 2' %}
+```scala
+import java.util.{Random => _, _}
+```
+
+{% endtab %}
+{% tab 'Scala 3' for=packaging-imports-9 %}
+
 ```scala
 import java.util.{Random as _, *}
 ```
+{% endtab %}
+{% endtabs %}
 
 If you try to access the `Random` class it won’t work, but you can access all other members from that package:
 
+{% tabs packaging-imports-10 %}
+{% tab 'Scala 2 and 3' %}
 ```scala
 val r = new Random   // won’t compile
 new ArrayList        // works
 ```
+{% endtab %}
+{% endtabs %}
 
 #### Hiding multiple members
 
 To hide multiple members during the import process, list them before using the final wildcard import:
 
+{% tabs packaging-imports-11 class=tabs-scala-version %}
+{% tab 'Scala 2' %}
+```scala
+scala> import java.util.{List => _, Map => _, Set => _, _}
```

Comment:
I see that there was a `scala>` prefix in the original Scala 3 snippet as well, but we should remove it:

```suggestion
import java.util.{List => _, Map => _, Set => _, _}
```